src/components/controls/dates/DatePicker/DatePicker.tsx

```diff
@@ -46,6 +46,9 @@ interface DatePickerBaseProps {
 
   /** `dayjs` input format. Default is `YYYY-MM-DD`. */
   inputFormat?: string
+
+  /** Allow free input of date */
```

Comment:
Let's be explicit about the default here?

```suggestion
  /** Whether to allow free input. Default is `false`. */
```